specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2020-10-26-preview/diagnosticServicesToken_API.json

```diff
@@ -0,0 +1,243 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApplicationInsightsManagementClient",
+    "description": "Azure Application Insights client for application insights diagnostic services token.",
+    "version": "2020-10-26-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Insights/operations": {
```

Comment:
I found a lot of input-files include this operation. It's a common used API, you should move it in a separated input-file and don't need to write anymore. What you should do is including that file in the releasing tag.

Reply:
By the way, a releasing tag should contain only one definition of "Operations_List". As I know, tag `package-preview-2020-06` contains multiple definitions of it.

Reply:
Infect this can cause confusion and possible issues document generation as you have two API with exactly same name/path

Reply:
Well, it seems like I followed the wrong example. Removed as suggested.